### ISSUE TITLE
Piped instance

### ISSUE THREAD

**Issue by newpipetube** [TeamNewPipe/NewPipe#8685]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Add piped instance

### Why do you want this feature?

Add piped instance

**Comment by mhmdanas** [comment]
It's not clear what you're asking for exactly. Please elaborate.

**Comment by newpipetube** [comment]
A new instance https://piped.kavin.rocks/
That doesnt submit data to youtube.

**Comment by mhmdanas** [comment]
Yes, I know that instance, but what do you want us to do with it exactly? Fetch data from it instead of YouTube?

**Comment by newpipetube** [comment]
Yes exactly. Currently theres youtube, peertube and bandcamp as choices in newpipe. Add piped to fetch data as giving data to youtube sucks.

**Comment by mhmdanas** [comment]
Thank you for the clarification. I edited your issue to make clear what you're requesting.
